mod_reforged/hooks/items/weapons/weapon.nut

```diff
@@ -1,5 +1,16 @@
+::Reforged.HooksMod.hookTree("scripts/items/weapons/weapon", function(q) {
+	q.create = @(__original) function()
+	{
+		__original();
+		if (this.m.Reach == null)
+		{
+			this.assignDefaultReach();
+		}
+	}
+});
```

Comment:
I think reach should default to a value and then be adjustable within individual create functions, so someone could do something like

```squirrel
this.m.Reach -= 1
```

Reply:
Best I can think of for this case is to set the Reach to a negative value by default instead of `null`. For example -20
That way we can respect mods, which do simple addition with it.

But who would do such operations on an item, which they don't even know about?
And if they did it on a itemtype basis, they would also use a `hookTree` which should fire after `Reforged`, since that mod builds on Reforged

Reply:
I can see the case for `null` or starting at 0.